### DISCUSSION TITLE
task warrior

### DISCUSSION THREAD

**Discussion by mosaleh-dev** [ransome1/sleek#334] [Ideas]
This UI  is very elegant, thanks for this contribution to the open source community 
 I'm using task warrior for task management , i do not like available options for GUI (there is none UNIX apps as far as i know) 
I do not know the time will it take but is this possible to support task warrior file format

**Comment by ransome1** [comment]
Hi @mosaleh52 . I'm afraid sleek is an app based solely on the todo.txt format. I'm afraid there are currently no plans to add further formats.

**Reply by mosaleh-dev** [reply]
hey @ransome1 
Thanks for your efforts making this awesome project , I will try to make convertor, so I can use sleek GUI with taskwarrior backend
